.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+on:
+    pull_request:
+    push:
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [14, 16, 18]
```

Comment:
Tests are timing out on Node 18 likely because the tests use localhost rather than 127.0.0.1. Replace localhost with 127.0.0.1 in connection strings and see if that fixes the tests